packages/fxa-settings/src/components/ConnectedServices/MOCK_SERVICES.tsx

```diff
@@ -0,0 +1,203 @@
+    approximateLastAccessTime: null,
+    approximateLastAccessTimeFormatted: null,
+  },
+  {
+    clientId: null,
+    deviceId: 'bf7c2ae1fa0ebd146f6478ac68bd8d63',
+    sessionTokenId:
+      '8ee9174a04f76d5ec5bf0c93bbd91b837796f067da408e58b5bd5d1fd914b45b',
+    refreshTokenId: null,
+    isCurrentSession: false,
+    deviceType: 'desktop',
+    name: 'vladikoff4’s Nightly on Vlads-MacBook-Pro',
+    createdTime: 1571151269728,
+    lastAccessTime: 1571151288620,
+    scope: null,
+    location: {
+      city: 'Toronto',
+      country: 'Canada',
+      state: 'Ontario',
+      stateCode: 'ON',
+    },
+    userAgent: 'Firefox 71',
+    os: 'Mac OS X',
+    createdTimeFormatted: 'a month ago',
+    lastAccessTimeFormatted: 'a month ago',
+    approximateLastAccessTime: null,
+    approximateLastAccessTimeFormatted: null,
+  },
+  {
+    clientId: '3c49430b43dfba77',
+    deviceId: '9b16949ac8d5019ea08fdc0b82731a97',
+    sessionTokenId: null,
+    refreshTokenId:
+      '081ada3fb68fc41c97a60358bf6443c13c55a2ea5a85b2cac0c4c2970e2d22f7',
+    isCurrentSession: false,
+    deviceType: 'mobile',
+    name: 'A-C Logins Sync Sample',
+    createdTime: 1570038359000,
+    lastAccessTime: 1570038359594,
+    scope: ['https://identity.mozilla.com/apps/oldsync', 'profile'],
+    location: {
+      city: null,
+      country: null,
+      state: null,
+      stateCode: null,
+    },
+    userAgent: '',
+    os: null,
+    createdTimeFormatted: 'a month ago',
+    lastAccessTimeFormatted: 'a month ago',
+    approximateLastAccessTime: null,
+    approximateLastAccessTimeFormatted: null,
+  },
+  {
+    clientId: '749818d3f2e7857f',
+    deviceId: null,
+    sessionTokenId: null,
+    refreshTokenId: null,
+    isCurrentSession: false,
+    deviceType: null,
+    name: 'Pocket',
+    createdTime: 1558476112000,
+    lastAccessTime: 1558476112000,
+    scope: ['profile:display_name', 'profile:email', 'profile:uid'],
+    location: {
+      city: null,
+      country: null,
+      state: null,
+      stateCode: null,
+    },
+    userAgent: '',
+    os: null,
+    createdTimeFormatted: '6 months ago',
+    lastAccessTimeFormatted: '6 months ago',
+    approximateLastAccessTime: null,
+    approximateLastAccessTimeFormatted: null,
+  },
+  {
+    clientId: '802d56ef2a9af9fa',
+    deviceId: null,
+    sessionTokenId: null,
+    refreshTokenId:
+      'e2e4001b8184721aba5cb7bf5df8f5af16d6f36dc144c1f40d9c86397cc83225',
+    isCurrentSession: false,
+    deviceType: null,
+    name: 'Firefox Monitor',
+    createdTime: 1556745521000,
+    lastAccessTime: 1556745521000,
+    scope: ['profile'],
+    location: {
+      city: null,
+      country: null,
+      state: null,
+      stateCode: null,
+    },
+    userAgent: '',
+    os: null,
+    createdTimeFormatted: '6 months ago',
+    lastAccessTimeFormatted: '6 months ago',
+    approximateLastAccessTime: null,
+    approximateLastAccessTimeFormatted: null,
+  },
+];
```

Comment:
Not a blocker, but it looks like a handful of these object keys are unchanging across each mock, and others are values, like timestamps and unique IDs, could be programmatically created (hooray for variability in tests!). You might consider turning these into a function that returns a mock.

Also, this doesn't need to be a TSX file 🙂

Reply:
@meandavejustice this doesn't need to be a TSX file

Reply:
fixed it up :)